lib/core/db_models/worker/schema/workerHistorySchema.dart

```diff
@@ -23,4 +23,11 @@ class WorkerHistorySchema {
   static const String occupations = "occupations";
   static const String googlePlaceId = "google_place_id";
   static const String googlePlaceData = "google_place_data";
+  static const String external = "external";
```

Comment:
external is a special word can we use this?

Reply:
i am using the key as provided by the backend developer

Reply:
this looks like a special protect key word, it wont complile!